ebuild-writing/functions/src_prepare/text.xml

```diff
@@ -48,8 +48,9 @@ src_prepare() {
 <body>
 <codesample lang="ebuild">
 src_prepare() {
-	epatch "${FILESDIR}/${PV}/${P}-fix-bogosity.patch"
-	use pam &amp;&amp; epatch "${FILESDIR}/${PV}/${P}-pam.patch"
+	default
```

Comment:
IMHO "default" doesn't make much sense unless patches are applied via the PATCHES variable. So I'd rather have an explicit eapply_user after the eapply calls.